sum/contracts/src/deploy.ts

```diff
@@ -23,35 +24,39 @@ const __filename = fileURLToPath(import.meta.url);
 const __dirname = path.dirname(path.dirname(__filename));
 
 (async () => {
-  let deployedInfo = await deploySC(
+  let deployed = await deploySC(
     publicApi,
     deployerAccount,
     [
       {
-        data: readFileSync(path.join(__dirname, 'build', 'main.wasm')),
-        coins: 0,
-        args: new Args(),
+        data: readFileSync(path.join(__dirname, 'build', 'sum.wasm')),
+        coins: fromMAS(0.5),
+        args: new Args([]),
```

Comment:
You should use noArgs instead

Reply:
I can't, NoArg is from as-types. If we want to do so, we should implement it in massa-web3